### DISCUSSION TITLE
`assertion failed [block != nullptr]: BasicBlock requested for unrecognized address` error when `pnpm prisma db push`

### DISCUSSION THREAD

**Discussion by mannerism** [prisma/prisma#20739] [Q&A] [answered]
### Question

# Development Environment
1. MacOS Ventura 13.4.1
2. MacBook Pro, 64 GB, Apple M1 Max, 2022
1. Tailwind, Trpc, Planetscale (cloud MySql), Typescript, pnpm

# Problem
Whenever I try to push new db schema using `pnpm prisma db push` command, I get the following error sporadically: 

```zsh
assertion failed [block != nullptr]: BasicBlock requested for unrecognized address
(BuilderBase.h:550 block_for_offset)
```

Based on my experience I can say it's about 60-70% of my try. If I try the same command several more times it works. 

The real problem associated with this error is this command will spin off `node` worker as a background process and won't go off until you release it manually using `Activity Monitor` or `CLI`. 

In other words, everytime I fail `pnpm prisma db push` it will spin a node instance and it accumulates overtime without noticing. Yesterday I found out that my pretty beefed-up M1 Max macbook pro reached 99% in CPU usage and found out that there were 8 node instances running in the background crowding the majority of CPU usage. 

Want to know more about this issue.

### Expected behavior (optional)

Running `pnpm prisma db push` without error and hanging behavior.

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by nurul3101** [comment, marked as answer]
Hey @mannerism 👋 

There is a related GitHub Issue for the same error you are running into, can you try this workaround?

- https://github.com/prisma/prisma/issues/17869#issuecomment-1428061665

**Reply by mannerism** [reply]
Thank You. I confirm it's node problem in my local environment.
